asciidoc/courses/aura-dashboards/modules/3-building-dashboards/lessons/1-first-card/lesson.adoc

```diff
@@ -0,0 +1,115 @@
+= First card
+:type: lesson
+:order: 1
+
+In this lesson, you will learn how to create your first card in Neo4j Aura Dashboards, using either the natural language AI assistant or by writing Cypher queries.
+
+You will explore the different card types available and understand how to visualise your graph data effectively: 
+
+* **Bar Chart**: Ideal for comparing quantities across categories.
+* **Line Chart**: Best for showing trends over time.
+* **Pie Chart**: Useful for displaying proportions of a whole.
+* **Table**: Great for detailed data representation.
+
+Optionally, you can dive into writing Cypher queries to fetch and visualise aggregated data, providing more control over the data displayed in your cards.
+
+== Connecting
+
+Before you start, make sure you have completed the previous lesson and imported the sample data into your Aura instance.
```

Comment:
Should we say _loaded the sample data_? Since the script doesn't use the _Import_ service? Or am I mistaken? Since I can't build this course, just reviewing it here..